example/App.tsx

```diff
@@ -335,13 +354,20 @@ function OpenLastExampleSetting() {
 interface MainScreenItemProps {
   name: string;
   onPressItem: (name: string) => void;
+  enabled: boolean;
 }
 
-function MainScreenItem({ name, onPressItem }: MainScreenItemProps) {
+function MainScreenItem({ name, onPressItem, enabled }: MainScreenItemProps) {
   return (
-    <RectButton style={[styles.button]} onPress={() => onPressItem(name)}>
+    <RectButton
+      enabled={enabled}
+      style={[
+        styles.button,
+        !enabled && { backgroundColor: 'rgba(220, 220, 220, 0.8)' },
```

Comment:
This looks like highlighting not disabling, can you make the text less opaque?

Reply:
Done in [91072ac](https://github.com/software-mansion/react-native-gesture-handler/pull/3135/commits/91072acb376ed50de69547d61f91695ca627bec6)